test/entrypoint/srv_ddl.lua

```diff
@@ -110,11 +111,50 @@ package.preload['customers-storage'] = function()
                 }
             }
 
+            -- DDL module doesn't support map and array types and JSON path in
+            -- sharding key, see https://github.com/tarantool/ddl/issues/81,
+            -- so we create space with jsonpath index manually.
+            local function create_space_with_jsonpath()
+                local customers_jsonpath_key = box.schema.space.create('customers_jsonpath_key', {
+                    format = {
+                        {name = 'id', is_nullable = false, type = 'map'},
+                        {name = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                        {name = 'name', is_nullable = false, type = 'string'},
+                        {name = 'age', is_nullable = false, type = 'number'},
+                        {name = 'data', is_nullable = false, type = 'map'},
+                    },
+                    if_not_exists = true,
+                    engine = engine,
+                })
+                customers_jsonpath_key:create_index('pk', {
+                    parts = {
+                        {1, 'unsigned', path = 'customer_id.unsigned'},
+                        {5, 'unsigned', path = 'customer.weight'},
+                    },
+                    if_not_exists = true,
+                })
+                customers_jsonpath_key:create_index('bucket_id', {
+                    parts = { 'bucket_id' },
+                    unique = false,
+                    if_not_exists = true,
+                })
+                box.space['_ddl_sharding_key']:insert({
+                        'customers_jsonpath_key',
+                        {'name'}
+                })
+            end
+
             if not box.info.ro then
                 local ok, err = ddl.set_schema(schema)
                 if not ok then
                     error(err)
                 end
+                if crud_utils.tarantool_supports_jsonpath_indexes() then
+                    local ok, err = pcall(create_space_with_jsonpath)
+                    if not ok then
+                        error(err)
+                    end
```

Comment:
Nit: pcall + error work the same as just call.

Reply:
Code is gone with removing tests for jsonpath as sharding key definition support.

Reply:
But I have fixed it in a branch for jsonpath as sharding key definition support:
```diff
--- a/test/entrypoint/srv_ddl.lua
+++ b/test/entrypoint/srv_ddl.lua
@@ -150,10 +150,7 @@ package.preload['customers-storage'] = function()
                     error(err)
                 end
                 if crud_utils.tarantool_supports_jsonpath_indexes() then
-                    local ok, err = pcall(create_space_with_jsonpath)
-                    if not ok then
-                        error(err)
-                    end
+                    create_space_with_jsonpath()
                 end
             end
```